src/main.rs

```diff
@@ -143,13 +143,20 @@ pub fn main() {
             let args = std::env::args().skip(2);
             if let Some(first) = target.kind.get(0) {
                 if target.kind.len() > 1 || first.ends_with("lib") {
+                    println!("compiling library");
```

Comment:
Right, but `debug!()` lets you log when you want to debug -- we don't want regular users to see this output